capi/README.md

```diff
@@ -21,10 +21,11 @@ pkg-config pdfflow --cflags
 pkg-config pdfflow --libs
 ```
 
-If you installed to a non-standard location, you need to set up the `PKG_CONFIG_PATH` and `LD_LIBRARY_PATH`, e.g.:
+If you installed to a non-standard location, you need to set up the `PKG_CONFIG_PATH` and `LD_LIBRARY_PATH`, e.g., for a `VIRTUAL_ENV`:
 ```bash
 export PKG_CONFIG_PATH=${VIRTUAL_ENV}/lib/pkgconfig/:${PKG_CONFIG_PATH}:
 export LD_LIBRARY_PATH=${VIRTUAL_ENV}/lib/:${LD_LIBRARY_PATH}:
+export DYLD_LIBRARY_PATH=${VIRTUAL_ENV}/lib:${DYLD_LIBRARY_PATH}Ñ
```

Comment:
Btw, this doesn't seem to work:
```
dyld[2128]: Library not loaded: @rpath/Python3.framework/Versions/3.9/Python3
  Referenced from: <7EC5A4F3-CC1E-3A1E-9CDB-F4C71F9EF443> /Users/carrazza/repo/n3pdf/pdfflow/capi/examples/example
  Reason: tried: '/Users/carrazza/repo/n3pdf/pdfflow/env/lib/Python3.framework/Versions/3.9/Python3' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Users/carrazza/repo/n3pdf/pdfflow/env/lib/Python3.framework/Versions/3.9/Python3' (no such file), '/Users/carrazza/repo/n3pdf/pdfflow/env/lib/Python3.framework/Versions/3.9/Python3' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Users/carrazza/repo/n3pdf/pdfflow/env/lib/Python3.framework/Versions/3.9/Python3' (no such file)
Abort trap: 6
```

Reply:
In order to get that type of error the user would probably be mixing the system python with brew (for cffi/cmake)

A possible solution is to tell the executable where to point.
```
install_name_tool -change @rpath/Python3.framework/Versions/3.9/Python3 /Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/libpython3.9.dylib example
```

but it enters in the same category as the `sys.executable`. It can be fixed automatically:

```
if(APPLE)
add_custom_command(TARGET pdfflow
    POST_BUILD COMMAND 
    ${CMAKE_INSTALL_NAME_TOOL} -change "@rpath/Python3.framework/Versions/3.9/Python3" ${Python3_LIBRARIES}
    $<TARGET_FILE:pdfflow>)
endif()
```

(and remove the `@pythonlibs@` from `pdfflow.pc.in` otherwise it will take the python library from there)

With those changes it works for me for the system python, using cmake from brew... but I insist that if someone is trying to do that, it's on them...